packages/web-components/src/components/masthead/masthead.scss

```diff
@@ -488,20 +490,9 @@ $breakpoint--desktop-nav: 799px;
 
 @mixin left-nav-overlay {
   position: fixed;
-  z-index: calc(#{z('header')} - 1);
+  z-index: #{z('header')} - 1;
   background-color: rgba(22, 22, 22, 0.5);
   block-size: 100vh;
-  clip-path: polygon(
```

Comment:
Just curious why this clip-path goes away?

Reply:
The short answer is that I couldn't find a good reason to keep it around. Our z-index values are doing its job, only better.

The long answer is kind of difficult to describe, but essentially it was trying to clip out an area for the open mobile menu so it would appear "above" the background overlay. This, combined with the new position rules, could cause situations where the background overlay would sit partially atop the global banner. I removed it to see what broke, and to my surprise, nothing appeared to. So I left the change in 🙂

Reply:
Nice. Yeah that checks out with what I eventually understood it to be. The clip path was like a carve out for the hambuger / close icon, which like you said, doesn't need to be that complicated. Nice clean up 💅

Reply:
Is the left-nav supposed to go full-width? Is the L0 supposed to go under the darkening overlay?

On prod:
<img width="677" alt="Screenshot 2024-05-29 at 11 36 55 AM" src="https://github.com/carbon-design-system/carbon-for-ibm-dotcom/assets/25532785/7e385dc5-719c-48f5-be4b-63c1377f28ae">

Deploy preview:
<img width="660" alt="Screenshot 2024-05-29 at 11 37 15 AM" src="https://github.com/carbon-design-system/carbon-for-ibm-dotcom/assets/25532785/4eaece75-fcd4-4b3a-9250-e4c427e5a657">

@areagan1030 can you sign off on this or do we need to fix some stuff?

Reply:
or @oliviaflory

Reply:
Specifically - the existing focus trapping keeps focus in the left-nav and the close button, and it cannot get to search/contact/user menu items on the right side of the L0.

Reply:
@andy-blum @oliviaflory I am looking back at our functional specs from the L0 work in Figma. This documentation matched the prod screenshot — outside of the close icon, the logo and utility items are dimmed, and the width of the menu appears to match the specs.

Reply:
@andy-blum Good catches. These are existing regressions, but I'm not sure when they made their way in. They're in the latest v2 release: https://codepen.io/jakaeser/pen/dyeJMzp.

Can you file a bug report so we can address in a separate issue?

Reply:
I reworked the clip-path to account for the new `--c4d-masthead-top-nav-menu-bottom-edge` property, and now the overlay sits atop the L0 bar when the mobile menu is open, as per the design specs.

Reply:
> Can you file a bug report so we can address in a separate issue?

Andy opened [ADCMS-5191](https://jsw.ibm.com/browse/ADCMS-5191) on our internal board for the focus trap issues.